db/migrate/20181011104937_add_discovered_to_api_docs_services.rb

```diff
@@ -0,0 +1,5 @@
+class AddDiscoveredToApiDocsServices < ActiveRecord::Migration
+  def change
+    add_column :api_docs_services, :discovered, :boolean, null: true
```

Comment:
I see your point, but `null: true` will prevent a hard migration on Saas and in the end we only need it to be truthy or falsy.

Reply:
1. No need to ping ops
2. If it is not null and we need a default then we need to ping ops

So keep the null true because rails will transform it to false